src/utility.py

```diff
@@ -31,16 +31,18 @@ def filter_records(data, field, value_list, logger, mask=False, exclusion_analys
     excluded_data = None
 
     # Filter records
-    if not mask:
-        logger.info(f"Selecting records that satisfy {field} in {value_list} from {original_records} records.")
-        if exclusion_analysis:
-            excluded_data = data.loc[~data[field].isin(value_list)]
-        data = data.loc[data[field].isin(value_list)]
-    else:
+    if mask:
+        # Excluding records that match the filter criteria
         logger.info(f"Selecting records that satisfy {field} not in {value_list} from {original_records} records.")
         if exclusion_analysis:
             excluded_data = data.loc[data[field].isin(value_list)]
         data = data.loc[~data[field].isin(value_list)]
+    else:
+        # Including records that match the filter criteria
+        logger.info(f"Selecting records that satisfy {field} in {value_list} from {original_records} records.")
+        if exclusion_analysis:
+            excluded_data = data.loc[~data[field].isin(value_list)]
+        data = data.loc[data[field].isin(value_list)]
```

Comment:
I think this was `if not mask` originally as mask will almost always be `False`, but this looks much more maintainable//readable

Reply:
I did wonder whether it was better style (easier to understand) if the meaning of `mask` was reversed so that the default remained including records that match the filter, but the value was `True`.